packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -382,20 +382,32 @@ class IosProject extends XcodeBasedProject {
     }
 
     ErrorHandlingFileSystem.deleteIfExists(ephemeralModuleDirectory, recursive: true);
+
+    final String? iosDevelopmentTeam = await getCodeSigningIdentityDevelopmentTeam(
```

Comment:
Really this should be cached somewhere but it's really hard to find a common object to hang it off of, or it would need to cached in a new object that gets piped through in a bunch of places that shouldn't need to care about iOS codesigning (to avoid a new `globals`).